karrot/groups/models.py

```diff
@@ -91,7 +95,7 @@ class Group(BaseModel, LocationModel, ConversationMixin, DirtyFieldsMixin):
         upload_to='group_photos',
         null=True,
     )
-    features = ArrayField(TextField(), default=list)
+    features = ArrayField(TextField(), default=default_group_features)
```

Comment:
Just wondering, does this function really get called on group creation?

Reply:
Yes, well I run it locally and created a group and it had offers enabled. I had noticed it because I needed a group with offers by my new local groups didn't have offers enabled.